Projects/Features/MainFeature/Sources/BroadcastCollectionViewModel.swift

```diff
@@ -69,14 +55,16 @@ public class BroadcastCollectionViewModel: ViewModel {
     }
     
     private func fetchData() {
-
         usecase.execute()
-            .sink { error in
-                print(error)
-            } receiveValue: { ids in
-                print(ids.first?.channelId)
-                self.output.items.send(ids.map { Item(title: $0.channelId, subtitle1: "test", subtitle2: "test") })
-            }.store(in: &cancellables)
+            .sink(
+                receiveCompletion: { _ in },
+                receiveValue: { channel in
+                    self.output.items.send(channel.map {
+                        Channel(title: $0.name, image: $0.image)
+                    })
+                }
+            )
+            .store(in: &cancellables)
```

Comment:
```suggestion
            .sink { channel in
                self.output.items.send(channel.map {
                    Channel(title: $0.name, image: $0.image)
                })
            }
            .store(in: &cancellables)
```

위 처럼 작성하면 더 깔끔할 것 같습니다.

Reply:
Publisher이기 때문에 제시하신것과 같이 작성이 힘들 것 같습니다!

<img width="750" src="https://github.com/user-attachments/assets/38bdce79-a4d2-456b-9600-bf24ecbbe647">